src/Makefile.test.include

```diff
@@ -232,341 +233,348 @@ test_test_bitcoin_LDADD += $(LIBBITCOIN_ZMQ) $(ZMQ_LIBS)
 endif
 
 if ENABLE_FUZZ
-test_fuzz_block_SOURCES = $(FUZZ_SUITE) test/fuzz/block.cpp
-test_fuzz_block_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
-test_fuzz_block_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
-test_fuzz_block_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
-test_fuzz_block_LDADD = $(FUZZ_SUITE_LD_COMMON)
 
-test_fuzz_block_deserialize_SOURCES = $(FUZZ_SUITE) test/fuzz/deserialize.cpp
-test_fuzz_block_deserialize_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) -DBLOCK_DESERIALIZE=1
-test_fuzz_block_deserialize_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
-test_fuzz_block_deserialize_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
-test_fuzz_block_deserialize_LDADD = $(FUZZ_SUITE_LD_COMMON)
```

Comment:
not sure if it makes sense to reorder the whole file to "reduce merge conflicts". This causes a merge conflict ...

Reply:
That is 100% intentional: it introduces _one_ merge conflict now but avoids more or less _all_ future merge conflicts by making it clear where to add new fuzzing harnesses. It makes sense given that clarification?

Reply:
Should sorting these be done in a separate MR to keep this one more focused?

Reply:
@paymog It is discouraged to submit this kind of changes as separate focused PRs. Changes like these should if possible made when touching the files for other purposes like in this case :)

Reply:
@paymog it's true that the [refactor section of CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#refactoring) seems to encourage making separate PRs, but in practice it seems to me that when the change is not really worth making its own PR and is trivial/non-controversial or made/requested by an regular contributor, making the change when touching relevant code can be a good option.

Reply:
I guess there's a deeper philosphical question of what makes a change worth a separate PR. When I work professionally I try to keep changes as focused as possible - though of course I'm not perfect.

Regarding non-controversiality, it seems that @MarcoFalke and @promag find this a bit controversial 😉. I don't want to push any ideas here since I'm still new to the community and I'm not sure there's much to gain either way in this instance - this change is pretty small. Making a new PR has overhead but might slightly reduce the review load.

Reply:
@paymog I was attempting to clarify the general case, not bikeshed. YMMV. I've found that there are reasons why things in this project might be somewhat different from one's professional work philosophy in other projects. Cheers.

Reply:
@jonatack I realize that my comment may not come across as I meant it. I didn't think you were trying to bikeshed and I'm not either - like I say I'm still new and learning how this community operates. I appreciate you trying to bring clarity. Hopefully I didn't offend you and if I did, I'm sorry. It wasn't my intention.

Reply:
No worries, I'm trying to grok things too and I make a lot of mistakes while doing it.